integration_test/support/models.exs

```diff
@@ -94,6 +104,8 @@ defmodule Ecto.Integration.Permalink do
     belongs_to :post, Ecto.Integration.Post
     has_many :post_comments_authors, through: [:post, :comments_authors]
   end
+
+  before_delete Ecto.Integration.Model, :callback_check, []
```

Comment:
I mean, the integration test  gurantees the data is removed, so we don't need to check for a callback.